target/linux/rtl838x/base-files/etc/board.d/01_leds

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+. /lib/functions/leds.sh
+. /lib/functions/uci-defaults.sh
+
+
+board=$(board_name)
+boardname="${board##*,}"
+
+board_config_update
+
+case $board in
+tp-link,t2500g-10ts)
```

Comment:
Please use "tplink" (without hyphen) in the board name
(https://www.kernel.org/doc/Documentation/devicetree/bindings/vendor-prefixes.txt)

Reply:
OK.